Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,98 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+}
+
+section{
+    padding: 100px 0;
+}
+
+.max-width{   
+    max-width: 1300px;
+    padding: 0 80px;
+    margin: auto;
+}
+
+.navbar{
+    padding-left: 30px;
+    position: fixed;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: #ffffff;
+}
+
+.home{
+    background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url("../images/flowershop.jpg");
```

Comment:
Very good use of linear-gradient to ensure a darker image!
For all of your other colours try to stick to a single colour convention: hexidecimal, rgba or a colour string.